requirements.txt

```diff
@@ -60,6 +60,6 @@ s3transfer==0.1.12
 six==1.11.0
 unicodecsv==0.14.1
 urllib3==1.22
-Werkzeug==0.12.2
+Werkzeug==0.13
```

Comment:
You'll enjoy this @galund - this got bumped by the pip freezing script. Seems our version of werkzeug is being dictated to us by our version of the apiclient :D